README.md

```diff
@@ -16,40 +16,51 @@
 We're writing an OpenAPI v3 authored specification for the current [Lob API](https://docs.lob.com/).
 This repo contains the spec as well as a growing set of tooling for working with OpenAPI v3 specs.
 
-## How the spec is organized
+## Design
 
-Our spec is organized semantically, by _resource_, instead of syntactically, by OpenAPI element.
+Our spec is a multifile spec organized semantically, by _resource_, instead of syntactically, by OpenAPI element. Organizing the spec semantically reduces cognitive friction, helping developers reason from _interaction_ (endpoints) to data (and process) design. As developers from multiple teams work with the spec, the design surfaces business semantics rather than presenting the canonical wall-o-yaml (or json-schema) that typifies the traditional API spec.
```

Comment:
I *think* I understand what this is getting at, but it's kind of dense for me and I think going into a little more detail about the difference between semantics and syntax might help with that. Maybe a compare-and-contrast?

Reply:
Ah, that's a good idea. I'm definitely planning on expanding the design section! Thanks for the suggestion.